packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/FileUtils.java

```diff
@@ -121,4 +128,8 @@ private static void copy(InputStream in, OutputStream out) throws IOException {
     }
     out.flush();
   }
+
+  private static String getBaseName(String fileName) {
+    return fileName.split("\\.")[0];
```

Comment:
The basename is everything before the last `.`, not the first, so this will return the wrong name in some cases.